### ISSUE TITLE
Come up with funny posts to type

### ISSUE THREAD

**Issue by carbonsam** [ChristianJHughes/js13kgames#18]

**Comment by carbonsam** [comment]
@ChristianJHughes What if we used one (or a few) ipsum generators? That would be pretty great! https://idsgn.dropmark.com/107

**Comment by ChristianJHughes** [comment]
@carbonsam Dank.

**Comment by carbonsam** [comment]
Samuel L. Ipsum is currently my favorite, but probably a bit too much for what we're going for 😂

**Comment by ChristianJHughes** [comment]
I got the Bob Ross L. Ipsum generator working, but dealing with multiple lines on the canvas is the worst 😢. I need to do some more work on that -- I think we'll want posts that are longer than a single line.

Also, Kontra doesn't support most punctuation (includes commas and periods). So we're stuck with strings of alphanumeric characters. Old documentation hasn't been updated, I went crazy debugging.
